### ISSUE TITLE
Nice Job. Could you please share the dataset and the training code?

### ISSUE THREAD

**Issue by ainrichman** [thepowerfuldeez/facemesh.pytorch#1]
This model performs very well in my testing. Could you please share the dataset and the training code?

**Comment by thepowerfuldeez** [comment]
Hi ainrichman! Sadly, but I don't have dataset and training code. Although, I used such model in my pipeline, I didn't train it, because I only ported pre-trained weights from Google MediaPipe repo: https://github.com/google/mediapipe/tree/master/mediapipe/models
They didn't release dataset, but stated:
"In our training, we rely on a globally sourced dataset of around 30K in-the-wild mobile camera photos taken from a wide variety of sensors in changing lighting conditions. During training, we further augment the dataset with standard cropping and image processing primitives, and also a few specialized ones: modelling camera sensor noise [8] and applying a randomized non-linear parametric transformation to the image intensity histogram (the latter helps simulating marginal lighting conditions)."
